spec/e2e/migrations_spec.rb

```diff
@@ -0,0 +1,113 @@
+describe Neo4j::Migrations::Runner do
```

Comment:
One trick that I've seen is that you can surround your `RSpec` with the module(s) in question to reduce the amount of repetition of those modules.  For example here you could do:

```ruby
module Neo4j
  module Migrations
    describe Runner do
      # etc...
      before do
        # etc...
        SchemaMigration.delete_all
      end
      # etc...
    end
  end
end
```